ClipboardPickle/tag-security-privacy.md

```diff
@@ -12,7 +12,7 @@ Yes, this information is necessary to enable the native apps to communicate with
 
 ### 2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
 
-Through the custom clipboard formats, PII may be transferable from web to native apps or vice versa. The risk is comparable to the existing DataTransfer APIs that expose unsanitized HTML content in the standard HTML format(via setData/getData methods), which is separate from custom formats, in that the native/web apps have to explicitly opt-in to read/write custom formats. Moreover, the custom format contents are less visible to the sites that don't know how to parse it, so this should alleviate some privacy concerns.
+Through the custom clipboard formats, PII may be transferable from web to native apps or vice versa. However, the content in the custom format is less visible/obvious to the users. This is also true for the existing DataTransfer APIs that expose unsanitized HTML content in the standard HTML format(via setData/getData methods), but there may be metadata present in the custom format that wouldn't be typically included in the HTML format. The parsing rules for the custom format content and what data is included in the format, have to be defined by the native and web apps that read/write this format, so that alleviates some privacy concerns regarding who can read the sensitive data (if present) in the custom formats.
```

Comment:
nit: is less visible/obvious -> may be less visible/obvious.

Reply:
Done.